habitat-lab/habitat/config/benchmark/rearrange/cat_nav_to_obj.yaml

```diff
@@ -31,9 +31,22 @@ habitat:
     auto_sleep: True
     agents:
       main_agent:
-        radius: 0.3
-        robot_urdf: ./data/robots/hab_fetch/robots/hab_suction.urdf
-        robot_type: FetchSuctionRobot
+        radius: 0.25
+        robot_urdf: data/robots/hab_stretch/urdf/hab_stretch.urdf
+        robot_type: "StretchRobot"
+        sim_sensors:
+           head_rgb_sensor:
+             height: 256 # 640
+             width: 256 # 480
+             hfov: 90 # 43
```

Comment:
This is actually 42 not 43, just change the comment so that you dont forget when you are moving back to the right configs